NMS-14162: docs on setting capabilities (#4529)
OpenNMS/opennms: docs/modules/deployment/pages/core/centos-rhel/initialize-core.adoc

sudo /opt/opennms/bin/install -dis
----

._Binding to privileged ports_

The core service user must be able to send ICMP echo requests.
During setup, the permissions for `net.ipv4.ping_group_range` are set permanently on boot via `/etc/sysctl.d/99-opennms-non-root-icmp.conf`.
Some system kernels do not honor this setting, and in those cases, additional configuration is required to allow binding to privileged ports.

> This is best achieved by creating a systemd overlay to add the necessary settings to the service configuration.

Reviewer: ```suggestion
We recommend creating a systemd overlay to add the necessary settings to the service configuration.
```
